src/pages/ProductList/components/ProductCards.tsx

```diff
@@ -9,6 +9,10 @@ interface ProductItemProps {
 }
 
 function ProductCards({ products = [] }: ProductItemProps) {
+  if (!products?.length) {
+    return <div>등록된 상품이 없습니다.</div>;
+  }
```

Comment:
```suggestion
  if (products == null) {
    return <Loader />
  }
  
  if (products?.length === 0) {
    return <div>등록된 상품이 없습니다.</div>;
  }
```
~~(마이너) Suspense를 처리해주고 있어서 문제는 없겠지만 falsy한 값을 다룰때 주의하자는 의미에서 코멘트를 남겨보아요 🙏 
products가 optional이 들어와도 true가 되어서 값을 처리할 때 products가 데이터 fetch가 되었고 개수가 0개일때를 정확히 조건식에 넣어주는걸 권장드려요~~

useProductList에서 null 병합연산으로 배열을 만들어주고 있어서 null이나 optional이 들어올 여지가 없었네요..!
ProductList 타입만 받게 변경하면 될 것 같아요